### ISSUE TITLE
I'm confused about the dim9,did the paper give a input dimension as 9?

### ISSUE THREAD

**Issue by LeopoldACC** [AnTao97/dgcnn.pytorch#46]
the code show as below
```python
def forward(self, x):
        batch_size = x.size(0)
        num_points = x.size(2)

        x = get_graph_feature(x, k=self.k, dim9=True)   # (batch_size, 9, num_points) -> (batch_size, 9*2, num_points, k)
```

**Comment by antao97** [comment]
DGCNN follows the setting of PointNet for semantic segmentation. In the `5.1. Applications` of the PointNet paper, the authors introduce the 9-dim vector for semantic segmentation. 

The description is presented as follows:

`To prepare training data, we firstly split points by room, and then sample rooms into blocks with area 1m by 1m. We train our segmentation version of PointNet to predict per point class in each block. Each point is represented by a 9-dim vector of XYZ, RGB and normalized location as to the room (from 0 to 1).`

**Comment by LeopoldACC** [comment]
thanks a lot!
